app/editor/src/components/form/formpage/FormPage.tsx

```diff
@@ -4,11 +4,15 @@ import * as styled from './styled';
 
 export interface IFormPageProps extends InputHTMLAttributes<HTMLDivElement> {
   /** bypass the default min width */
-  bypassMinWidth?: boolean;
+  minWidth?: string;
```

Comment:
I've never liked the boolean property.  Now to override, just set it to an empty string, or a specific size.